rflx/specification/grammar.py

```diff
@@ -427,13 +427,11 @@ def checksum_aspect() -> Token:
 def message_type_definition() -> Token:
     first_aspect = Keyword("First") - Keyword("=>") - mathematical_expression(restricted=True)
     first_aspect.setParseAction(parse_aspect)
-    length_aspect = Keyword("Length") - Keyword("=>") - mathematical_expression(restricted=True)
-    length_aspect.setParseAction(parse_aspect)
 
     then = locatedExpr(
         Keyword("then")
         - (Keyword("null") | unqualified_identifier())
-        - Group(Optional(with_aspects(delimitedList(first_aspect | length_aspect))))
+        - Group(Optional(with_aspects(delimitedList(first_aspect | size_aspect()))))
```

Comment:
Are the added parentheses intended?

Reply:
Yes, `size_aspect` is a function which is also used for the size aspect of range integers.